openpype/hosts/fusion/plugins/publish/validate_existing_frames_existence.py

```diff
@@ -0,0 +1,83 @@
+import os
+import pyblish.api
+
+from openpype.pipeline.publish import RepairAction
+from openpype.pipeline import PublishValidationError
+
+from openpype.hosts.fusion.api.action import SelectInvalidAction
+
+
+class ValidateLocalFramesExistence(pyblish.api.InstancePlugin):
+    """Checks if files for savers that's set
+    to publish existing frames exists
+    """
+
+    order = pyblish.api.ValidatorOrder
+    label = "Validate Existing Frames Exists"
+    families = ["render"]
+    hosts = ["fusion"]
+    actions = [RepairAction, SelectInvalidAction]
+
+    @classmethod
+    def get_invalid(cls, instance, non_existing_frames=None):
+        if non_existing_frames is None:
+            non_existing_frames = []
+
+        active = instance.data.get("active", instance.data.get("publish"))
+        if not active:
+            return []
```

Comment:
This shouldn't run on an inactive instance as far as I know. If it does, that's a bug elsewhere.

Reply:
It's actually code taken from `validate_create_folder_checked.py` so I took it as a needed part.
I just tried to comment that part out and it works as expected without that part. Gonna remove it from the code.